realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -195,15 +195,36 @@ public static Table initTable(ImplicitTransaction transaction) {
     }
 
     public static void validateTable(ImplicitTransaction transaction) {
-        if(transaction.hasTable("class_AllTypes")) {
+        if (transaction.hasTable("class_AllTypes")) {
             Table table = transaction.getTable("class_AllTypes");
-            if(table.getColumnCount() != 9) {
+
+            if (table.getColumnCount() != 9) {
                 throw new IllegalStateException("Column count does not match");
```

Comment:
Perhaps rephrase to "Number of fields does not match the expected. Is x, expected y"?